admin-ui/plugins/fido/components/StaticConfiguration.tsx

```diff
@@ -13,50 +13,57 @@ import GluuProperties from 'Routes/Apps/Gluu/GluuProperties'
 import { adminUiFeatures } from 'Plugins/admin/helper/utils'
 
 import { validationSchema, transformToFormValues, fidoConstants } from '../helper'
-
-function StaticConfiguration({ fidoConfiguration, handleSubmit }) {
-  const staticConfiguration = fidoConfiguration.fido.fido2Configuration
-
+import {
+  StaticConfigurationProps,
+  StaticConfigurationFormValues,
+  CredentialTypeOption,
+  RequestedPartyOption,
+} from './types/StaticConfiguration'
+import type { AppConfiguration, FormData } from '../types'
+
+function StaticConfiguration({
+  fidoConfiguration,
+  handleSubmit,
+}: StaticConfigurationProps): JSX.Element {
   const { t } = useTranslation()
-
-  const [modal, setModal] = useState(false)
-
-  const toggle = useCallback(() => {
+  const [modal, setModal] = useState<boolean>(false)
+  const toggle = useCallback((): void => {
     setModal((prev) => !prev)
   }, [])
-
-  const formik = useFormik({
-    initialValues: transformToFormValues(staticConfiguration, fidoConstants.STATIC),
+  const formik = useFormik<StaticConfigurationFormValues>({
+    initialValues: transformToFormValues(
+      fidoConfiguration.fido as AppConfiguration,
+      fidoConstants.STATIC,
+    ) as StaticConfigurationFormValues,
     onSubmit: toggle,
-    validationSchema: validationSchema[fidoConstants.VALIDATION_SCHEMAS.STATIC_CONFIG],
+    validationSchema: validationSchema.staticConfigValidationSchema,
   })
```

Comment:
Formik should reinitialize when config arrives

The config is fetched async; without enableReinitialize the form keeps empty defaults after the API response.

Apply:

```diff
   const formik = useFormik<StaticConfigurationFormValues>({
     initialValues: transformToFormValues(
       fidoConfiguration.fido as AppConfiguration,
       fidoConstants.STATIC,
     ) as StaticConfigurationFormValues,
     onSubmit: toggle,
     validationSchema: validationSchema.staticConfigValidationSchema,
+    enableReinitialize: true,
   })
```